docs/en/ingest-management/elastic-agent/scaling-on-kubernetes.asciidoc

```diff
@@ -0,0 +1,290 @@
+[[scaling-on-kubernetes]]
+= Scaling {agent} on kubernetes
+
+For more information on how to deploy Elastic Agent on Kubernetes please review the pages below:
+
+- How to run the {agent} on <<running-on-kubernetes-managed-by-fleet>>.
+- How to run the {agent} on <<running-on-kubernetes-standalone>>.
```

Comment:
```suggestion
- <<running-on-kubernetes-managed-by-fleet>>.
- <<running-on-kubernetes-standalone>>.
```

Reply:
^^ I believe that this is what you're going for. When you use an internal link (`<<section-id>>`) without additional text, you get the topic title subbed in for the link. 

With current formatting, you're getting this: 

<img width="783" alt="Screen Shot 2023-06-12 at 5 17 41 PM" src="https://github.com/elastic/ingest-docs/assets/35154725/8e3bc753-427f-4c22-ae94-cb334af59e7c">